projects/knora-ui/src/lib/viewer/pipes/knoradate.pipe.ts

```diff
@@ -7,25 +7,33 @@ import { KnoraDate } from '@knora/api';
 export class KnoraDatePipe implements PipeTransform {
 
     // TODO: use the optional format parameter and return a string with that format
-    transform(value: KnoraDate, format?: string): string {
-        if (!(value instanceof KnoraDate)) {
+    transform(date: KnoraDate, format?: string): string {
+        if (!(date instanceof KnoraDate)) {
             console.error('Non-KnoraDate provided. Expected a valid KnoraDate');
             return '';
         }
 
-        if (value == null) {
+        if (!date) {
```

Comment:
Can this condition ever be met? If a `date` is not a `KnoraDate`, it will already have left the function on line 13.

Reply:
true, it'll never be met. Removed in e52f86d